financial-connections/src/main/java/com/stripe/android/financialconnections/repository/FinancialConnectionsAccountsRepository.kt

```diff
@@ -130,7 +134,7 @@ private class FinancialConnectionsAccountsRepositoryImpl(
     ): NetworkedAccountsList {
         val request = apiRequestFactory.createGet(
             url = networkedAccountsUrl,
-            options = apiOptions,
+            options = provideConsumerApiOptions() ?: apiOptions,
```

Comment:
Potential unification alternative (happy to discuss!):

- `provideConsumerApiOptions` -> `provideApiOptions(isConsumerCall: Boolean)` or similar.
- We inject the regular `ApiOptions` on this `provideApiOptions` component, and move the logic on this line into there.
- We stop injecting `ApiOptions` in repositories and always use the provider you built.

WDYT?

Reply:
Yep, I think that makes sense. Let me play around with that!